urlcache.go

```diff
@@ -80,20 +80,23 @@ func (c *urlcache) keepCurrent(currentDate time.Time) {
 		headResp, err := http.Head(c.url)
```

Comment:
If the server supports it, we might as well use GET. What is this being used for again?